conjure/conjure_test.go

```diff
@@ -852,6 +852,30 @@ func (u *ExampleUnion) AcceptFuncs(strFunc func(string) error, otherFunc func(st
 	}
 }
 
+func (u *ExampleUnion) StrNoopSuccess(string) error {
+	return nil
+}
+
+func (u *ExampleUnion) OtherNoopSuccess(string) error {
```

Comment:
Decided to make these methods rather than static functions to avoid name collisions or requiring the union name within the name